apps/vmq_server/src/vmq_queue.erl

```diff
@@ -520,6 +527,25 @@ handle_sync_event(get_sessions, _From, StateName, #state{sessions=Sessions} = St
     {reply, maps:keys(Sessions), StateName, State};
 handle_sync_event(get_opts, _From, StateName, #state{opts=Opts} = State) ->
     {reply, Opts, StateName, State};
+handle_sync_event({force_disconnect, DoCleanup}, _From, StateName,
+                  #state{id=SId, sessions=Sessions, offline=#queue{queue=OfflineQ}} = State) ->
+    %% Forces to disconnect all sessions and to cleanup all state
```

Comment:
I think this would read nicer like this: `Forcefully disconnect all sessions and cleanup all state`